README.md

```diff
@@ -54,9 +54,19 @@ npm install -g oav
 ```
 bash-3.2$ oav -h
 Commands:
-  validate-example <spec-path>  Performs validation of x-ms-examples and
-                                examples present in the spec.
-  validate-spec <spec-path>     Performs semantic validation of the spec.
+  extract-xmsexamples <spec-path>           Extracts the x-ms-examples for a
+  <recordings>                              given swagger from the .NET session
+                                            recordings and saves them in a file.
+  generate-wireformat <spec-path>           Transforms the x-ms-examples for a
+                                            given operation into raw
+                                            request/response format and saves
+                                            them in a markdown file.
+  resolve-spec <spec-path>                  Resolves the swagger spec based on
```

Comment:
How about `Resolves the swagger spec based on the selected options like allOfs, relativePaths, examples etc.` ?